spec/index.html

```diff
@@ -10369,7 +10369,151 @@ <h3>Evaluation Semantics</h3>
+            
+              <p>This process is applied throughout the graph pattern of <code>EXISTS</code>:</p>
+
+              <div class="defn">
+                <b>Definition: <span id="defn_varrename" name="defn_varrename">Variable Remapping</span></b>
+                <p>
+                  For any algebra expression `X`, define the Variable Remapping `PrjMap(X)`of algebra expression `X`:
+                </p>
+                  <pre>PrjMap(X) = replace all project operations Project(P, PV) 
+                with ProjectMap(P,PV) for each projection in X.</pre>
+              </div>
+              <p>
+                The outcome of `PrjMap` is independent of the order of replacement
+                (e.g. bottom-up or top-down).
+                Replacements may happen several times, depending on recusive order
+                but each time a replacement is made, the variable not used anywhere else.
+              </p>
+        
+              <p class="note">
+                A variable inside a project expression that is not in the variables projected
+                is not affected by the values insertion operation because it is renamed apart.
+              </p>
+            </section>
+
+            <section>
+              <h4>Values Insertion</h4>
+              <div class="defn">
+                <div>
+                  <b>Definition: <span id="defn_valuesinsertion" name="defn_valuesinsertion">Values Insertion</span></b>
+                  <p>
+                    Define the Values Insertion function `ValuesInsert(X, μ)`
+                  </p>
+                  <pre>Let Table(μ) = { μ } and multiplicity( μ | Table(μ) = { μ } ) = 1
+
+Replace each occurence of `Y` in X where `Y` is one of
+          <a href="#sparqlTranslateBasicGraphPatterns">Basic Graph Pattern</a>,
+          <a href="#sparqlTranslatePathExpressions">Property Path Expression</a>,
+          <a href="#sparqlTranslateGraphPatterns">`Graph(Var, pattern)`</a>,
+          <a href="#https://www.w3.org/TR/sparql12-query/#sparqlTranslateGraphPatterns">Inline Data</a>
+with `join(Y, Table(μ))`.</pre>
+
+                </div>
+                <p>@@ rename as ???correllate</p>
+              </div>
+
+              <div class="example">
+                <p>
+                  Examples
+                </p>
+              </div>
+            </section>
+
+            <section>
+              <h4>Evaluation of EXISTS</h4>
+              <div class="defn">
+                <b>Definition: <span id="x-defn_evalExists" name="x-defn_evalExists">Evaluation of Exists</span></b>
+                <p>
+                  Let `μ` be the current solution mapping for a filter, and `X` a graph pattern,
+                      define the Evaluation of Exists `exists(X)`
+                </p>
+                <pre>exists(X) = true 
+              if eval( D(G), ValuesInsert(PrjMap(X), μ) )
+                 is a non-empty solution sequence.
+exists(X) = false otherwise</pre>
```

Comment:
```suggestion
Otherwise, exists(X) = false.</pre>
```

Reply:
The style here is that for function definition, with each case being aligned. It is not running text.